modules/users/client/components/TopNavigationSmall.component.js

```diff
@@ -0,0 +1,173 @@
+  }
+
+  const [showRemoveModal, setShowRemoveModal] = useState(false);
+
+  const { t } = useTranslation('user');
+
+  const isSelf = selfId === userId;
+
+  let links;
+  if (isSelf) {
+    links = [
+      {
+        id: 'edit-profile',
+        label: t('Edit your profile'),
+        link: '/profile/edit',
+      },
+    ];
+  } else {
+    links = [
+      {
+        id: 'send-message',
+        label: t('Send a message'),
+        link: `/messages/${username}`,
+      },
+    ];
+
+    if (referencesEnabled) {
+      links.push({
+        id: 'write-reference',
+        label: t('Write a reference'),
+        link: `/profile/${username}/references/new`,
+      });
+    }
+
+    if (isResolved) {
+      const contactLink = !contact._id
+        ? {
+            id: 'add-contact',
+            label: t('Add contact'),
+            link: `/contact-add/${userId}`,
+          }
+        : contact.confirmed
+        ? {
+            id: 'remove-contact',
+            label: t('Remove contact'),
+            tooltip: t('Contacts since {{created, MMM DD, YYYY}}', {
+              created: new Date(contact.created),
+            }),
+            onClick: () => setShowRemoveModal(true),
+          }
+        : {
+            id: 'delete-contact-request',
+            label: t('Delete contact request'),
+            tooltip: t('Request sent {{created, MMM DD, YYYY}}', {
+              created: new Date(contact.created),
+            }),
+            onClick: () => setShowRemoveModal(true),
+          };
+
+      links.push(contactLink);
+    }
+  }
+
+  function handleRemoveSuccess() {
+    setShowRemoveModal(false);
+
+    // broadcast the removal to angular
+    onContactRemoved(contact);
+  }
+
+  return (
+    <>
+      {contact && (
+        <RemoveContact
+          selfId={selfId}
+          contact={contact}
+          show={showRemoveModal}
+          onSuccess={handleRemoveSuccess}
+          onCancel={() => setShowRemoveModal(false)}
+        />
+      )}
+      <nav className="navbar navbar-white navbar-fixed-top navbar-fixed-top-below visible-xs-block">
+        <div className="container">
+          <ul className="nav navbar-nav" role="navigation">
+            {links.map(({ id, label, link, tooltip, onClick }) => (
+              <li key={id}>
+                <NavButton
+                  id={id}
+                  label={label}
+                  link={link}
+                  tooltip={tooltip}
+                  onClick={onClick}
+                />
+              </li>
+            ))}
+          </ul>
+        </div>
+      </nav>
+    </>
+  );
+}
+
+TopNavigationSmall.propTypes = {
+  username: PropTypes.string.isRequired,
+  selfId: PropTypes.string.isRequired,
+  userId: PropTypes.string.isRequired,
+  contact: PropTypes.object,
+  referencesEnabled: PropTypes.bool.isRequired,
+  isResolved: PropTypes.bool.isRequired,
+  onContactRemoved: PropTypes.func.isRequired,
+};
+
+function LabelWithTooltip({ id, label, tooltip }) {
```

Comment:
I think it's fine to leave tooltips out from mobile navigation; they never get shown since there's no _hover_ on mobile devices.

<img width="236" alt="image" src="https://user-images.githubusercontent.com/87168/76560974-08ea9d00-64ab-11ea-81ff-9798ddbd2aec.png">

That'll simplify this a bit.

Reply:
Is the small screen unlikely to be used on desktop?
Also, the tooltip is present in the original code. i.e. this change will degrade/simplify the original.
@simison If you're happy with the above points, I'll proceed with the change. I would appreciate a confirmation.

Reply:
I think we can consider everything under 768px as a touch screen, and very unlikely used on a desktop.

If it was on original code it's my bad. 😅 Let's clean out now that we have a chance while looking into it.

Reply:
Removed the tooltip in ad97da390e46b965c2a789f0d030eb87dd361388.